int64 num1
int64 num2
---
int64 sum
int64 product
bool are_equal